doc/cascadia/AddASetting.md

```diff
@@ -380,8 +382,13 @@ OBSERVABLE_PROJECTED_SETTING(Microsoft.Terminal.Settings.Model.CloseOnExitMode,
 // Profiles.h --> ProfileViewModel
 // - this defines the setting as observable off of the _profile object
 OBSERVABLE_PROJECTED_SETTING(_profile, CloseOnExit);
+
+// Profiles.h --> ProfileViewModel
+// - if the setting cannot be altered by the user, use the following macro instead:

```

Comment:
this is incorrect. it wouldn't need to be a setting if it weren't alterable.

this is for settings that cannot be CLEARed